Turkey/PlatformId.cs

```diff
@@ -63,25 +65,27 @@ public List<string> ComputePlatformIds(string[] osReleaseLines, string architect
             return platforms.ToList();
         }
 
-        private string GetValue(string key, string[] lines)
+        private static string GetValue(string key, string[] lines)
         {
             return lines.Where(line => line.StartsWith(key + "=", StringComparison.Ordinal)).Last().Substring((key + "=").Length);
         }
 
-        private string Unquote(string text)
+        private static string Unquote(string text)
         {
             // TODO implement proper un-escaping
             // This is a limited shell-style syntax described at
             // https://www.freedesktop.org/software/systemd/man/os-release.html
-            if (text.StartsWith("\"") && text.EndsWith("\""))
+            if (text.StartsWith("\"", StringComparison.Ordinal) && text.EndsWith("\"", StringComparison.Ordinal))
             {
                 return text.Substring(1, text.Length - 2);
             }
 
             return text;
         }
 
+#pragma warning disable CA1822 // Mark members as static
         internal string GetLddVersion()
```

Comment:
Other methods are now `static`, so I am okay with marking this as `static` too.

Reply:
Setting this to static causes an error when executing `dotnet test -f net6.0 -c Release --verbosity detailed Turkey.Tests`

Error: `Turkey.Tests/PlatformIdTest.cs(34,30): error CS0176: Member 'PlatformId.GetLddVersion()' cannot be accessed with an instance reference; qualify it with a type name instead`